test/Optimizer/testsimplepathbrfold.baseline

```diff
@@ -1,6 +1,5 @@
 100
 TRACE PathDependentBranchFolding: Can prove retarget of branch in Block 1 from Block 2 to Block 4 in func foo
-TRACE PathDependentBranchFolding: Can prove retarget of branch in Block 2 from Block 4 to Block 3 in func foo
 100
 Done foo
 

```

Comment:
This trace changed because we are doing this opt after OptBlock now, so the branch was optimized away and didn't need folding.